{{cookiecutter.plugin_name}}/{{cookiecutter.module_name}}/__init__.py

```diff
@@ -0,0 +1,32 @@
+from importlib_metadata import version, PackageNotFoundError
+
+from flask import Blueprint
+
+
+__version__ = "Unknown version"
```

Comment:
This should move to before the `import` statements (except `from __future__ ` statements). See https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names.